### ISSUE TITLE
[A11y] Advanced Search: Headings

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1795]
### Operating System

Windows

### Browser

Edge

### Affected page

Advanced Search

### Admin user?

- [X] Yes

### Issue Description

Headings should be reworked:

- There are two level 1 headings on the page, neither of which actually describe the purpose of the page (advanced search).  There's "Oregon Digital (Development)" and "More Search Options".
- There are many level 3 headings.  The two that announce the area of the form make some amount of sense.  The headings that expand to checkboxes are bad, however.  They are not really acting like headings for content; they're more acting like a fieldset should.  Furthermore they're level 3 when the thing that describes them is already level 3.

### Suggested Fix

- Make a single level 1 heading that matches the function of the page
- Make "Find items that match <any/all> of" and "AND have these attributes" level 2 headings instead of level 3
- The facet headings could continue to be level 3 if they need to be headings.

Mentioned in slack, but just to repeat: this issue probably shouldn't be considered on its own.  I believe there will be other issues related to the way the facets work, and the whole picture will need to be considered for remediation, not just ticket-by-ticket.

**Comment by jechols** [comment]
Also: "Search Tips" acts as a heading of sorts even though it's not large text.  It might not fail WCAG, I'm not sure.  But it probably should be coded as a heading just for usability.

**Comment by jechols** [comment]
Sent the "fieldset vs. headings" question to the webaim list.

**Comment by jechols** [comment]
See https://webaim.org/discussion/mail_message?id=46469

**Comment by jechols** [comment]
Answer for the fieldset vs. heading thing is the same as #1818: https://github.com/OregonDigital/OD2/issues/1818#issuecomment-898558179

**Comment by jechols** [comment]
Returning for work:

- The level 1 heading is now "More Search Options   Start Over", which doesn't match the function of the page (I think "Advanced Search" or even just "Search" would make more sense here)
- I believe that the "Search tips" piece should indeed be a heading; I'm not sure why I said it might not fail WCAG. It's visually presented as a heading, so it needs to be coded that way.

Similar to #1834: the facet accordions are almost perfect, but need this:

> The accordion header button element has aria-controls set to the ID of the element containing the accordion panel content.
